test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -60,12 +60,24 @@ public void listCommand_parsedCorrectly() {
     }
     
     @Test
-    public void sortCommand_parsedCorrectly() {
+    public void sortCommand_noArgs_parsedCorrectly() {
```

Comment:
You have successfully implemented Unit tests!

But you still need to add IO tests for your feature.
IO tests help assess the end-to-end correctness of a program.
You will need to add input.txt and expected.txt files along with a batch script to compare your programs output to the expected output.